api/CanvasPattern.json

```diff
@@ -55,16 +55,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasPattern/setTransform",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "67"
```

Comment:
This actually [landed in 68](https://storage.googleapis.com/chromium-find-releases-static/934.html#9347a40943840072196586f5a9e0839ba99375c1). Please change this `chrome_android`, and 'webview_android`.